lib/options.json

```diff
@@ -205,13 +206,16 @@
           ]
         }
       },
+      "description": "https://webpack.js.org/configuration/dev-server/#devserverclient",
       "additionalProperties": false
     },
     "compress": {
-      "type": "boolean"
+      "type": "boolean",
+      "description": "https://webpack.js.org/configuration/dev-server/#devservercompress"
     },
     "dev": {
-      "type": "object"
+      "type": "object",
+      "description": "https://webpack.js.org/configuration/dev-server/#devserverdev-"
```

Comment:
I think we should use proper descriptions instead of links. Links are subject to change as we improve docs.

Reply:
yeah we could but these show up in validation errors which can give more visibility of the error on the detailed docs page, I do have proper desc in older commit we can revert to that if we want /cc @alexander-akait

Reply:
We need keep description and link(s), only links is bad, especial for local usage, I don't want to open link for simple description

Reply:
yep lets do it

Reply:
done